### PR TITLE
fix: 🐛 upgrade to styled-components 5

### PR DESCRIPTION
**Pull request by robineng**
Maybe WIP. I haven't had time to fully test this when integrated with ui-mutations and webapp-next yet but the storybook seems to work.

### CODE REVIEW

File: src/Atoms/Media/__snapshots__/Media.stories.storyshot [hunk 1]
```diff
@@ -46,23 +70,47 @@ Array [
   }
 }
 
+@media not all and (min-width:976px) and (max-width:1279px) {
```
Comment (shmidt-i) [inline src/Atoms/Media/__snapshots__/Media.stories.storyshot]: These are weird, they are just empty

File: src/Atoms/Skeleton/__snapshots__/Skeleton.stories.storyshot [hunk 2]
```diff
@@ -142,6 +142,10 @@ Array [
   align-self: center;
 }
 
+@media (min-width:760px) {
```
Comment (mjmonline) [inline src/Atoms/Skeleton/__snapshots__/Skeleton.stories.storyshot]: empty media queries in this file as well

File: src/Molecules/Drawer/__snapshots__/Drawer.stories.storyshot [hunk 3]
```diff
@@ -234,12 +234,20 @@ exports[`Storyshots Molecules | Drawer Default Story 1`] = `
...
 
 @media (min-width:760px) {
-  .c2 {
+  .c3 {
     font-size: 24px;
     line-height: 1.1666666666666667;
   }
 }
 
+@media (min-width:760px) {
```
Comment (mjmonline) [inline src/Molecules/Drawer/__snapshots__/Drawer.stories.storyshot]: empty media queries in this file as well

File: src/Molecules/Input/Textarea/__snapshots__/Textarea.stories.storyshot [hunk 4]
```diff
@@ -1607,6 +1607,10 @@ Array [
   align-self: center;
 }
 
+@media (min-width:760px) {
+
```
Comment (mjmonline) [inline src/Molecules/Input/Textarea/__snapshots__/Textarea.stories.storyshot]: empty media queries in this file as well

File: src/Molecules/Modal/__snapshots__/Modal.stories.storyshot [hunk 5]
```diff
@@ -133,11 +133,19 @@ exports[`Storyshots Molecules | Modal Default Story 1`] = `
...
 
 @media (min-width:760px) {
-  .c1 {
+  .c2 {
     background-color: rgba(110,110,105,0.63);
   }
 }
 
+@media (max-width:759px) {
+
```
Comment (mjmonline) [inline src/Molecules/Modal/__snapshots__/Modal.stories.storyshot]: empty media queries in this file as well

File: src/Molecules/Number/__snapshots__/Number.stories.storyshot [hunk 6]
```diff
@@ -1279,9 +1270,26 @@ Array [
...
+  font-size: 14px;
+  line-height: 1.4285714285714286;
+}
+
+.c3 {
   font-weight: inherit;
 }
 
+@media (min-width:760px) {
+
```
Comment (mjmonline) [inline src/Molecules/Number/__snapshots__/Number.stories.storyshot]: this file as well

File: src/Molecules/Button/__snapshots__/Button.stories.storyshot [hunk 7]
```diff
@@ -1467,6 +1467,10 @@ Array [
   align-self: center;
 }
 
+@media (min-width:760px) {
+
```
Comment (mjmonline) [inline src/Molecules/Button/__snapshots__/Button.stories.storyshot]: this file as well

File: src/Molecules/Input/Checkbox/__snapshots__/Checkbox.stories.storyshot [hunk 8]
```diff
@@ -438,6 +412,10 @@ Array [
   align-self: center;
 }
 
+@media (min-width:760px) {
+
```
Comment (mjmonline) [inline src/Molecules/Input/Checkbox/__snapshots__/Checkbox.stories.storyshot]: this file as well

File: src/Molecules/Input/Number/__snapshots__/Number.stories.storyshot [hunk 9]
```diff
@@ -854,6 +854,10 @@ Array [
   align-self: center;
 }
 
+@media (min-width:760px) {
+
```
Comment (mjmonline) [inline src/Molecules/Input/Number/__snapshots__/Number.stories.storyshot]: this file as well

File: src/Molecules/Input/Radio/__snapshots__/Radio.stories.storyshot [hunk 10]
```diff
@@ -1340,6 +1280,10 @@ Array [
   align-self: center;
 }
 
+@media (min-width:760px) {
+
```
Comment (mjmonline) [inline src/Molecules/Input/Radio/__snapshots__/Radio.stories.storyshot]: this file as well

File: src/Molecules/Input/Text/__snapshots__/Text.stories.storyshot [hunk 11]
```diff
@@ -2147,6 +2151,10 @@ Array [
   margin-bottom: 32px;
 }
 
+@media (min-width:760px) {
+
```
Comment (mjmonline) [inline src/Molecules/Input/Text/__snapshots__/Text.stories.storyshot]: this file as well

File: src/Organisms/CardWithTabs/__snapshots__/CardWithTabs.stories.storyshot [hunk 12]
```diff
@@ -2,21 +2,21 @@
 
 exports[`Storyshots Organisms | CardWithTabs Controlled Behaviour 1`] = `
 Array [
-  <button
+  @media (min-width:760px) {
+
```
Comment (mjmonline) [inline src/Organisms/CardWithTabs/__snapshots__/CardWithTabs.stories.storyshot]: this file as well

**Comment by robineng** [comment]
@mjmonline @shmidt-i There seem to be others getting weird snapshots too so it might be related to jest-styled-components: https://github.com/styled-components/jest-styled-components/issues/276

**Pull request merged**